### PR TITLE
Insere post do Webnar #2 - Qlikview

### PR DESCRIPTION
**Pull request by hslinhares**
Insere post do Webnar #20 - Qlikview

### CODE REVIEW

**Review by carloshob — CHANGES_REQUESTED** [review]
@hslinhares, ficou muito bom seu post!

File: docs/blog/posts/20241108_webnar_qlikview.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+authors: [hslinhares]
+draft: false
+comments: true
+categories:
+  - Webnar Inteligência.MG
+---
+
+# Webnar Inteligência.MG #2 - Qlikview
+
+A ferramenta de BI `Qlikview` tem sido utilizada há vários anos na SPLOR em painéis de dados de grande importância para a execução das atividades, como o painel da `Reestimativa` e o `Relatório Operacional`, que trazem tempestividade e confiabilidade no acesso a informação.
```
[the remark this answers is not in the document]
Comment (hslinhares) [reply docs/blog/posts/20241108_webnar_qlikview.md:12]: Resolvido em [e7e1a55](https://github.com/splor-mg/handbook/pull/17/commits/e7e1a5509c68e7190d341b518b4c6aeefd666a89)

**Review by hslinhares — COMMENTED** [review]
Resolvido em [e7e1a55](https://github.com/splor-mg/handbook/pull/17/commits/e7e1a5509c68e7190d341b518b4c6aeefd666a89)

**Review by carloshob — APPROVED** [review]
Ficou ótimo!

**Pull request merged**